internal/credential/vault/credential_store.go

```diff
@@ -25,17 +24,6 @@ type CredentialStore struct {
 // client cert, namespace, TLS server name, and TLS skip verify are the
 // only valid options. All other options are ignored.
 func NewCredentialStore(scopeId string, vaultAddress string, token []byte, opt ...Option) (*CredentialStore, error) {
-	const op = "vault.NewCredentialStore"
```

Comment:
Why are these checks dropped?

Reply:
The checks are dropped because they are only required when persisting in storage (In the Create* call) but not when creating a credential store in memory (for Update for example).  The Create call already checks these values for being set.